src/styles/brackets_fonts.less

```diff
@@ -30,7 +30,11 @@
 /* SourceSansPro Regular */
 @font-face {
     font-family: 'SourceSansPro';
-    src: url('fonts/SourceSansPro/SourceSansPro-Regular.ttf');
+    src: local('Source Sans Pro Regular'),
+         url('fonts/SourceSansPro/SourceSansPro-Regular.eot?iefix') format('embedded-opentype'),
```

Comment:
ideally this goes the way of the dodo - if IE9< is not a concern, strip it as fast as possible =) It might then also make sense to simply not bother with the .ttf files, since everything that isn't IE supports .otf (even Android 4.2! If a little broken, it hates GSUB tables for instance), and the latest IEs support WOFF just fine.